File: java/Print adjacency list.java
class Solution {
    public List<List<Integer>> printGraph(int V, int edges[][]) {
        List<List<Integer>> list = new ArrayList<>();
        
        for(int i=0; i<V; i++){
            list.add(new ArrayList<>());
        }
        
        for(int i=0; i<edges.length; i++){
            list.get(edges[i][0]).add(edges[i][1]);
            list.get(edges[i][1]).add(edges[i][0]);
        }
        
        return list;
    }
}

File: cpp/Print adjacency list.cpp
class Solution {
  public:
    // Function to return the adjacency list for each vertex.
    vector<vector<int>> printGraph(int V, vector<pair<int, int>> edges) {
        vector<vector<int>> adjList(V);

        for (int i = 0; i < edges.size(); i++) {
            int u = edges[i].first;
            int v = edges[i].second;
            
            adjList[u].push_back(v);
            adjList[v].push_back(u);
        }

        return adjList;
    }
};

